### DISCUSSION TITLE
upgrade to 11.1.0.1 results in blank pages

### DISCUSSION THREAD

**Discussion by sbeier23** [arendst/Tasmota#15543] [Q&A] [answered]
Hello,

i currently have a few issues after an OTA upgrade from 11.0.0.3 to 11.1.0.1. Two different smartplugs got only blank pages via http access after upgrade (one Teckin and one Gosund SP111). When inspecting the sourcecode of the page ill get the following (sorry its the german firmware). Anyone got the same issues and has an idea to reflash the firmware (preferable without opening the case ;-)) 

`<!DOCTYPE html><html lang="de" class=""><head><meta charset='utf-8'><meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=no"/><link rel="icon" href="data:image/x-icon;base64,AAABAAEAEBACAAEAAQCwAAAAFgAAACgAAAAQAAAAIAAAAAEAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA////AP5/b+H6X2/h8k9v4eZnb+Hud2/h7ndv4e53b+FmZm/hMkxv4ZgZb+HOc2/h5+dv4fPPb+H5n2/h/D9v4f5/b+EAAO4EAADuBAAA7gQAAO4EAADuBAAA7gQAAO4EAADuBAAA7gQAAO4EAADuBAAA7gQAAO4EAADuBAAA7gQAAO4E"><title>Geschirrspueler - Hauptmenü</title><script>var x=null,lt,to,tp,pc='';eb=s=>document.getElementById(s);qs=s=>document.querySelector(s);sp=i=>eb(i).type=(eb(i).type==='text'?'password':'text');wl=f=>window.addEventListener('load',f);var ft;function la(p){a=p||'';clearTimeout(ft);clearTimeout(lt);if(x!=null){x.abort()}x=new XMLHttpRequest();x.onreadystatechange=()=>{if(x.readyState==4&&x.status==200){var s=x.responseText.replace(/{t}/g,"<table style='width:100%'>").replace(/{s}/g,"<tr><th>").replace(/{m}/g,"</th><td style='width:20px;white-space:nowrap'>").replace(/{e}/g,"</td></tr>");eb('l1').innerHTML=s;clearTimeout(ft);clearTimeout(lt);lt=setTimeout(la,2345);}};x.open('GET','.?m=1'+a,true);x.send();ft=setTimeout(la,2e4);}function lc(v,i,p){if(eb('s')){if(v=='h'||v=='d'){var sl=eb('sl4').value;eb('s').style.background='linear-gradient(to right,rgb('+sl+'%,'+sl+'%,'+sl+'%),hsl('+eb('sl2').value+',100%,50%))';}}la('&'+v+i+'='+p);}wl(la);function jd(){var t=0,i=document.querySelectorAll('input,button,textarea,select');while(i.length>=t){if(i[t]){i[t]['name']=(i[t].hasAttribute('id')&&(!i[t].hasAttribute('name')))?i[t]['id']:i[t]['name'];}t++;}}function sf(s){var t=0,i=document.querySelectorAll('.hf');while(i.length>=t){if(i[t]){i[t].style.display=s?'block':'none';}t++;}}wl(jd);<style>div,fieldset,input,select{padding:5px;font-size:1em;}fieldset{background:#4f4f4f;}p{margin:0.5em 0;}input{width:100%;box-sizing:border-box;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;background:#dddddd;color:#000000;}input[type=checkbox],input[type=radio]{width:1em;margin-right:6px;vertical-align:-1px;}input[type=range]{width:99%;}select{width:100%;background:#dddddd;color:#000000;}textarea{resize:vertical;width:98%;height:318px;padding:5px;overflow:auto;background:#1f1f1f;color:#65c115;}body{text-align:center;font-family:verdana,sans-serif;background:#252525;}td{padding:0px;}button{border:0;border-radius:0.3rem;background:#1fa3ec;color:#faffff;line-height:2.4rem;font-size:1.2rem;width:100%;-webkit-transition-duration:0.4s;transition-duration:0.4s;cursor:pointer;}button:hover{background:#0e70a4;}.bred{background:#d43535;}.bred:hover{background:#931f1f;}.bgrn{background:#47c266;}.bgrn:hover{background:#5aaf6f;}a{color:#1fa3ec;text-decoration:none;}.p{float:left;text-align:left;}.q{float:right;text-align:right;}.r{border-radius:0.3em;padding:2px;margin:6px 2px;}.hf{display:none;}</style></head><body><div style='text-align:left;display:inline-block;color:#eaeaea;min-width:340px;'><div style='text-align:center;color:#eaeaea;'><noscript>JavaScript aktivieren um Tasmota benutzen zu können<br></noscript><h3>Gosund SP111 2</h3><h2>Geschirrspueler</h2></div><div style='padding:0;' id='l1' name='l1'></div><table style='width:100%'><tr><td style='width:100%'><button onclick='la("&o=1");'>An/Aus</button></td></tr><tr></tr></table><div id=but3d style="display: block;"></div><p><form id=but3 style="display: block;" action='cn' method='get'><button>Einstellungen</button></form></p><p><form id=but4 style="display: block;" action='in' method='get'><button>Informationen</button></form></p><p><form id=but5 style="display: block;" action='up' method='get'><button>Firmware Update</button></form></p><p><form id=but14 style="display: block;" action='cs' method='get'><button>Konsole</button></form></p><p><form id=but0 style="display: block;" action='.' method='get' onsubmit='return confirm("Wirklich neustarten?");'><button name='rst' class='button bred'>Neustart</button></form></p><div style='text-align:right;font-size:11px;'><hr/><a href='https://bit.ly/tasmota' target='_blank' style='color:#aaa;'>Tasmota 11.1.0.1 von Theo Arends</a></div></div></body></html>`

I already tried different browsers. When accessing it, my username and password i set is requested and accepted.

Thanks in advance, Stefan

**Comment by sfromis** [comment, marked as answer]
Being fixed right now :-)

**Reply by s-hadinger** [reply]
Fixed now. Thanks for reporting.

**Reply by sfromis** [reply]
Precompiled binaries should get the update within an hour.

**Reply by sfromis** [reply]
When the precompiled binaries have gotten the fix, you should be able to upgrade using an URL like this in your browser:
`http://tasmota-device/cm?cmnd=Backlog+OtaUrl+http%3A%2F%2Fota.tasmota.com%2Ftasmota%2Ftasmota.bin.gz%3BUpgrade+1`

**Reply by sfromis** [reply]
Fixed now ....

**Reply by sbeier23** [reply]
Guys, Thank you so much. Already recovered one of the two devices. Other seems to have some wireless issues, but this is a local issue of myself. Thank you!!!
